x/protorev/keeper/developer_fees_test.go

```diff
@@ -0,0 +1,168 @@
+package keeper_test
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/osmosis-labs/osmosis/v13/app/apptesting"
+	"github.com/osmosis-labs/osmosis/v13/x/protorev/types"
+)
+
+// TestSendDeveloperFeesToDeveloperAccoutn tests the SendDeveloperFeesToDeveloperAccount function
+func (suite *KeeperTestSuite) TestSendDeveloperFeesToDeveloperAccount() {
+	cases := []struct {
+		description   string
+		malleate      func()
+		expectedErr   bool
+		expectedCoins sdk.Coins
+	}{
+		{
+			description:   "Send with unset developer account",
+			malleate:      func() {},
+			expectedErr:   true,
+			expectedCoins: sdk.NewCoins(),
+		},
+		{
+			description: "Send with set developer account",
+			malleate: func() {
```

Comment:
The structure of this test field looks very repetitive i.e. we're creating randomAcc, settingDevAccount, minting coins etc. Maybe we can use a helper setup function?

Reply:
added `pseudoExecuteTrade` as a wrapper to the functionality described above